gmailer_oauth.py

```diff
@@ -34,6 +34,10 @@ def create_message_with_attachment(to, subject, message_text, attachment):
     message = MIMEMultipart('related')
     message['to'] = to
     message['subject'] = subject
+    if cc:
```

Comment:
Since `cc` and `bcc` default to `None` we can remove the checks here and just have `message['cc'] = cc`